### PR TITLE
Back link integration tests

### PR DESCRIPTION
**Pull request by bogdan-zaharia**
**What**  
Added an integration test for back link.

**Why**  
To make sure it works as intended.

**Anything else?**  
 - Have you added any new third-party libraries? No.
 - Are any new environment variables or configuration values needed? No.
 - Anything else in this PR that isn't covered by the what/why? Yes. Disabled the "Finish & save" button when no vulnerabilities/assets are selected and when there are no notes.

### CODE REVIEW

File: cypress/integration/features/editSnapshot.spec.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ context('Edit snapshot', () => {
       vulnerabilities: [],
       id: '1'
     });
-
+    cy.visit(`/snapshots/1`);
```
Comment (kosiakkatrina) [inline cypress/integration/features/editSnapshot.spec.js]: Nice!

File: cypress/integration/features/editSnapshot.spec.js [hunk 2]
```diff
@@ -100,4 +97,16 @@ context('Edit snapshot', () => {
...
   });
+
+  describe('Back button', () => {
+    it('Sends the user back to Single View', () => {
+      cy.get('[data-testid=back-link-test]')
+        .should('contain', 'Back to Single View')
+        .and(
+          'have.attr',
+          'href',
+          'https://staging-singleview.hackney.gov.uk/customers/wub/view'
```
Comment (kosiakkatrina) [inline cypress/integration/features/editSnapshot.spec.js:116]: Looks good. Are we only saving SV ID in system IDs and if so why is it an array? Just a general question

File: pages/snapshots/[id].js [hunk 3]
```diff
@@ -74,6 +85,7 @@ const SnapshotSummary = ({ initialSnapshot, token }) => {
               await updateSnapshot(snapshot);
               setEditSnapshot(false);
             }}
+            disabled={!hasValue}
```
Comment (kosiakkatrina) [inline pages/snapshots/[id].js:88]: Nice!

File: cypress/integration/features/editSnapshot.spec.js (hunk 2 above)
Comment (mmmmillar) [reply cypress/integration/features/editSnapshot.spec.js:116]: Nah we're saving multiple ids

**Review by mmmmillar — APPROVED** [review]

**Pull request merged**